config/samples/apim_v1alpha1_ratelimitpolicy.yaml

```diff
@@ -5,30 +5,22 @@ metadata:
   name: toystore
 spec:
   rules:
-    - name: get-toy
-      operation:
-        paths: ["/toy"]
-        methods: ["GET"]
-        hosts: ["*.toystore.com"]
+    - operations:
```

Comment:
`name` has been deleted. 

There is a name in the next rules item. Do we still need a name?

Reply:
name is optional which is used to fetch operations from the VirtualService.